documentation/theme_1/theme1_template.md

```diff
@@ -8,5 +8,5 @@ You can fill in the placeholders with your own information to create a complete
 
```

Comment:
5. In a lean/agile work environment wasting time is a major thing we don't want to happen. Nice example of a theme.